src/views/Login.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+const Login = () => {
+  return (
+    <div>
+      <div className="">
+        <h4 className="center">Login</h4>
+        <form id="login">
+          <label for="email">Email</label>
```

Comment:
According to jsx, should this not be 'htmlFor'` instead of 'for`. Kindly check the two occurences